tests/pageObjects/LoginPage.js

```diff
@@ -0,0 +1,22 @@
+class LoginPage {
+    constructor(){
+        this.loginBtnLocator = ".btn-block";
+        this.emailFieldLocator = "#spree_user_email";
+        this.passwordFieldLocator = "#spree_user_password";
+        this.figureElementLocator = "(//figure//img)[2]";
```

Comment:
What are we locating with it?
Please name it accordingly
```suggestion
        this.figureElementLocator = "(//figure//img)[2]";
```